redaction/draft-strad-trans-redaction.md

```diff
@@ -69,26 +75,133 @@ certificates.
 TODO: Highlight better the differences between registered domains and
 subdomains, referencing the relevant DNS RFCs.
 
-Section TBD of [I-D.ietf-trans-rfc6962-bis] proposes two mechanisms for dealing
-with this conundrum: wildcard certificates and name-constrained intermediate
-CAs. However, these mechanisms are insufficient to cover all use cases.
-
-TODO(eranm): Expand on when each of the other mechanisms is suitable and when
-this mechanism may be suitable.
-
-We define a domain label redaction mechanism that covers all use cases, at the
-cost of increased implementation complexity. CAs and domain owners should note
-that there are privacy considerations ({{privacy_considerations}}) and that
-TLS clients may apply additional requirements (relating to the use of this
-redaction mechanism) for a certificate to be considered compliant.
-
 # Requirements Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC2119].
 
-# Redacting Labels in Precertificates    {#redacting_labels}
+# Redaction Mechanisms
+
+We propose three mechanisms to deal with this conundrum, in increasing order of
+implementation complexity:
+
+* Using wildcard certificates ({{wildcard_certificates}}) is the simplest
+  option, but it only covers certain use cases.
+
+* Logging a name-constrained intermediate CA certificate in place of the
+  end-entity certificate ({{name_constrained}}) covers more, but not all, use
+  cases.
+
+* Therefore, we define a domain label redaction mechanism ({{redacting_labels}})
+  that covers all use cases, at the cost of considerably increased
+  implementation complexity.
+
+We anticipate that TLS clients may develop policies that impose additional
+compliancy requirements on the use of the {{name_constrained}} and
+{{redacting_labels}} mechanisms.
```

Comment:
I don't think there are any restrictions on wildcard certs worth mentioning here.

AFAIK, there is no absolute "cannot have a wildcard for a domain in the PSL" rule at the moment.  The BRs do flag this as something for CAs to watch out for, but they also say:
"A CA is not prohibited from issuing a Wildcard Certificate to the Registrant of an entire gTLD, provided that control of the entire namespace is demonstrated in an appropriate way." (BRs v1.4.1 section 3.2.2.6)

The EV Guidelines prohibit Wildcard EV certs, but we already mention this in the "Using Wildcard Certificates" subsection.  (Also, this is an industry guideline rule rather than a TLS client policy rule).